source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -33,9 +33,6 @@ namespace HttpFilters {
 namespace Oauth2 {
 
 namespace {
-Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>
```

Comment:
What is the motivation for removing this? The inline lookups makes for slightly better perf than the regular header lookups

Reply:
Fixed